ibm/data_source_ibm_cos_bucket.go

```diff
@@ -186,6 +186,44 @@ func dataSourceIBMCosBucket() *schema.Resource {
 					},
 				},
 			},
+			"retention_rule": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1000,
+				Description: "A retention policy is enabled at the IBM Cloud Object Storage bucket level. Minimum, maximum and default retention period are defined by this policy and apply to all objects in the bucket.",
+				ForceNew:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
already the higher level attribute indicates its retention do we still need default_retention (_retention) at the end of each
I think default, minimum maximun permanent shd be good enough

Reply:
default in **testAccCheckIBMCosBucket_retention()** function taking as a keyword in **resource_ibm_cos_bucket_test.go** . So I kept it as it is.